### DISCUSSION TITLE
Question related to changes in PnP Management Shell registration in Microsoft 365

### DISCUSSION THREAD

**Discussion by matsla** [pnp/powershell#4208] [General]
Changes in PnP Management Shell registration in Microsoft 365
https://pnp.github.io/blog/post/changes-pnp-management-shell-registration

_Any existing script which is dependent on the multi-tenant authentication option will be impacted on this change starting from September 9, 2024. Please confirm your authentication model from your script(s)._

When using Azure Functions with PnP PowerShell (ex. 2.4.0) using Managed Identity or Entra App registration using certificate with application permission this will continue to work after September 9 without any changes needed?

This will only be an issue in scripts after 9 September when using connect with -Interactive ?

**Comment by gautamdsheth** [comment]
Yep, only impacts if you are using -interactive or credentials using PnP multi tenant app.

No changes needed if using certs or managed identity or your own EntraID app

**Reply by matsla** [reply]
Thanks @gautamdsheth , great news :)  We have 30+ customer implementations using PnP PowerShell + Azure Function and to update all customers in this short period would not work.

I also assume you don't need to update PNP PowerShell version in the case, it will still work if use ex. 2.4.0 after 9 sep. ?
